### PR TITLE
Upgrades and consistent formatting

### PR DESCRIPTION
**Pull request by fommil**

### CODE REVIEW

**Comment by fommil** [comment]
@ktonga

File: project/scalafmt.conf [hunk 1]
```diff
@@ -1,3 +1,4 @@
+maxColumn = 120
```
Comment (hrhino) [inline project/scalafmt.conf:1]: .............

**Review by hrhino — APPROVED** [review]
LGTM once you get Travis happy.

(Sundry test sources and build.sbt are ill-fmted)

**Comment by ktonga** [comment]
This looks as a good start although I recall seeing more differences when I applied the patch. I think we are missing the tests. Dunno if lines that were wrapped would rejoin or we need some sort of forcing flag for that to happen. Also we should turn off type tagging for vals in scalafix, that causes a lot of conflicts with code from sz8, mostly in the tests.

File: project/scalafmt.conf (hunk 1 above)
Comment (fommil) [reply project/scalafmt.conf:1]: don't get me started. You see what a mess it's making of by-name?

LOOK WHAT YOU MADE ME DO!

**Comment by fommil** [comment]
@ktonga maybe we should forward port the explicit types. We'd have to go through and remove them manually.

**Comment by ktonga** [comment]
TBH annotating the tests with `MatchResult` seems useless and unfamiliar. I can do it as part of the next packport while i'm in merging party.

**Review by jmcardon — APPROVED** [review]
I for one welcome our new 120 col length overlords

**Comment by hrhino** [comment]
@jmcardon confirmed for non-pureboi

**Review by NeQuissimus — APPROVED** [review]

**Pull request merged**